doc/components/adder.md

```diff
@@ -48,16 +48,16 @@ Here is an example of instantiating a `ParallelPrefixAdder`:
     print('${sum.value.toBigInt()}');
 ```
 
-## One's Complement Adder Subtractor
+## Ones Complement Adder Subtractor
```

Comment:
why remove apostrophe?
https://en.wikipedia.org/wiki/Two%27s_complement
https://en.wikipedia.org/wiki/Ones%27_complement

Reply:
I am moving to using ones' and twos' -complement throughout.